runbooks/workflow/payments/add_agency_dashboard_data_source.md

```diff
@@ -0,0 +1,190 @@
+    filter_value = '[agency-name]',
+    principals = ['serviceAccount:`agency-name`-payments-user@cal-itp-data-infra.iam.gserviceaccount.com']
+) }};
+```
+
+Substitute the following fields with the appropriate information for the agency that you are adding:
+
+- `filter_value` which is the Littlepay `participant_id` for the agency
+- `principals` which is the email address for the service account that was created in step #1. You can simply subsitute the agency name as used in that step as opposed to updating the whole string.
+
+Open a PR in Github and merge these changes. If you'd like access to the results of this policy before the next time the `transform_warehouse` DAG is run, you will need to run it manually. To do this, you should trigger the DAG with a selector [as described in the README for the DAG task](https://github.com/cal-itp/data-infra/tree/main/airflow/dags/transform_warehouse). Use selector: `{"dbt_select": "models/mart/payments"}`.
+
+### Add a new `Database` in Metabase for the agency
+
+**Permissions needed**: Member of the Metabase `Administrators` user group
+
+This creates the limited-access connection to the BigQuery warehouse.
+
+1. Navigate to Metabase, then `Settings`
+
+- In the upper-right hand corner, select the `Settings` wheel icon. Within the dropdown, select `Admin settings`.
+- In the top menu, select the `Databases` section to add a new databse
+- Select `Add database` in the top-right. Replace the following information:
+  - `Database type` --> BigQuery
+  - `Display name` --> `Payments - [agency name]`
+  - `Service account JSON file` --> upload the file downloaded in the previous section
+  - `Datasets` --> `Only these...`
+  - `Comma separated names of datasets that should appear in Metabase` --> `mart_payments`
+- `Save` your changes
+
+2. Add a new user `Group` for the agency and add users to the `Group`
+
+This step will limit the users that can access the previously created database.
+
+To begin, navigate to `People` in the top menu bar while within the `Admin settings` section
+
+#### Create a new user `Group`
+
+- Select `Groups` in the left-hand menu
+- Select `Create a group` and input `Payments Group - [agency name]`
+
+#### Add users to this new group
+
+- **If they have already have Metabase user accounts**:
+  - From within the `Groups` section, select the group that was just created
+  - From there, select `Add members` from the top-right of the page
+  - Type their name in the prompt, and once it auto-populates select `Add`
+- **If they don't have Metabase user accounts**:
+  - Select `People` in the left-hand menu
+  - Select `Invite someone` from the top-right of the page
+  - Populate `First name`, `Last name`, `Email`
+  - In the `Groups` dropdown, select the new agency group that was just created
+
+3. Create a new `Collection` for the agency
+
+This is a folder for the agency within Metabase where we will store their payments dashboard and the questions that comprise it
```

Comment:
Split into two sentences, e.g. "This is a folder for the agency within Metabase. We will store the agency's payments dashboard and the questions that comprise it in this folder"